### PR TITLE
Heroku changes

### PR DESCRIPTION
**Pull request by Kartihan-sri**
This is the latest heroku code changes. Setup is changed to run everything together, everything will be forked to https://github.com/Kartihan-sri/team-hazelnut and the deployed app can be viewed on https://lovingsitter.herokuapp.com/

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -17,13 +17,18 @@ import paymentRouter from "./routes/payment";
 import conversationRouter from "./routes/conversation";
 
 var app = express();
+const path = require("path")
```
Comment (Kartihan-sri) [inline app.js:20]: @ykurata In our previous code we had 
`var socket_io = require( "socket.io" );
var io = socket_io();
app.io = io;
const path = require("path")`

Do we not need the rest of this now?

Comment (ykurata) [reply app.js:20]: I think we don't need that part in app.js

File: app.js [hunk 2]
```diff
@@ -17,13 +17,18 @@ import paymentRouter from "./routes/payment";
...
 app.use(json());
 app.use(urlencoded({ extended: false }));
 app.use(cookieParser());
 app.use(cors());
-app.use(express.static(join(__dirname, "public")));
+app.use(express.static(path.join(__dirname, "client", "build")));
+
+app.get("*", (req, res) => {
+  res.sendFile(path.join(__dirname, "client", "build"));
+});
```
Comment (CometS1) [inline app.js]: I believe you need to remove this part for the get requests to work

**Review by CometS1 — CHANGES_REQUESTED** [review]

File: app.js (hunk 2 above)
Comment (Kartihan-sri) [reply app.js]: Removing the last bit of this code got the get requests to work as it should. I also noticed removing this whole code works as well, so I'll just delete this code

**Pull request merged**